### PR TITLE
Clean release notes

### PR DESCRIPTION
**Pull request by ela-kotulska-frequenz**

### CODE REVIEW

**Review by leandro-lucarella-frequenz — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,32 +2,16 @@
 
 ## Summary
 
+<!-- Here goes a general summary of what this release is about -->
+
 ## Upgrading
```
Comment (shsms) [inline RELEASE_NOTES.md:7]: Can you remove the Upgrading section?  Now that we have the "Bug fixes" section, we don't need this anymore.

Comment (shsms) [reply RELEASE_NOTES.md:7]: Oh already merged.  Nvm then.

Comment (ela-kotulska-frequenz) [reply RELEASE_NOTES.md:7]: Next time :)

Comment (shsms) [reply RELEASE_NOTES.md:7]: And should probably be renamed to `Breaking Changes`, not deleted.

**Pull request merged**